libparsec/crates/client/src/workspace/store/resolve_path.rs

```diff
@@ -27,6 +27,24 @@ pub(crate) enum PathConfinementPoint {
     Confined(VlobID),
 }
 
+impl From<PathConfinementPoint> for Option<VlobID> {
+    fn from(val: PathConfinementPoint) -> Self {
+        match val {
+            PathConfinementPoint::None => None,
```

Comment:
Could it be possible to rename the variant `None` to `NotConfined` ?

Reply:
done